OpenRA.Mods.RA/Traits/GpsWatcher.cs

```diff
@@ -48,30 +48,26 @@ public GpsWatcher(Player owner)
 		public void GpsRemove(Actor atek)
 		{
 			actors.Remove(atek);
-			RefreshGps(atek);
+			RefreshGps(atek.Owner);
```

Comment:
Can we please move this `.Owner` up a level? (i.e. pass the `Player launcher` into the method instead of `Actor atek`).